compiler/rustc_expand/src/expand.rs

```diff
@@ -1192,7 +1188,6 @@ impl<'a, 'b> MutVisitor for InvocationCollector<'a, 'b> {
     }
 
     fn visit_pat(&mut self, pat: &mut P<ast::Pat>) {
-        self.cfg.configure_pat(pat);
```

Comment:
I can't find a `configure!` call that configures patterns.

Reply:
Patterns themselves do not support attributes (including `cfg` attributes).
`configure_pat` only configures fields in `PatKind::Struct` which are already `configure!`d in `fn flat_map_field_pattern`.